src/Jupyter/Visualization/StateDisplayOperations.cs

```diff
@@ -46,6 +46,11 @@ public class JupyterDisplayDumper : QuantumSimulator.StateDumper
         /// </summary>
         public BasisStateLabelingConvention BasisStateLabelingConvention { get; set; } = BasisStateLabelingConvention.Bitstring;
 
+        /// <summary>
+        ///     Whether the measurement probabilities will be displayed as an
+        ///     interactive histogram.
+        /// </summary>
+        private bool ShowMeasurementDisplayHistogram { get; set; } = false;
```

Comment:
Does this imply that users need to explicitly opt-in with `%config` to get the new functionality? If so, where should we document that? We should likely also adapt samples as part of this to set that `%config` option.

Reply:
Well, this particular `bool` doesn't actually have that effect. This one simply gets set inside `Configure()` below with whatever the value of `dump.measurementDisplayHistogram` is. So the initial `false` value here does not impact the behavior.

The current default of `dump.measurementDisplayHistogram` is indeed `false`, though, as documented at https://docs.microsoft.com/qsharp/api/iqsharp-magic/config. As you point out, I believe we should flip this to `true` when ready (i.e. once the visualization features have been more fully finished). For the last QDK release I kept it at `false` in order to minimize potential impact on the release until we had done further testing.

Reply:
> Well, this particular `bool` doesn't actually have that effect. This one simply gets set inside `Configure()` below with whatever the value of `dump.measurementDisplayHistogram` is. So the initial `false` value here does not impact the behavior.

Ah, my apologies for misunderstanding!

> The current default of `dump.measurementDisplayHistogram` is indeed `false`, though, as documented at https://docs.microsoft.com/qsharp/api/iqsharp-magic/config. As you point out, I believe we should flip this to `true` when ready (i.e. once the visualization features have been more fully finished). For the last QDK release I kept it at `false` in order to minimize potential impact on the release until we had done further testing.

Makes perfect sense, thanks for the context!